css/navbar.css

```diff
@@ -1,27 +1,60 @@
-nav ul {
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
+nav {
+  margin-top: 0;
+}
+
+.topnav {
   overflow: hidden;
   background-color: #1D1283;
 }
 
-nav li {
-  float: left;
+.topnav div {
+  width: 30px;
+  height: 3px;
+  background-color: #E20665;
+  margin: 4px 0;
 }
 
-nav li a {
+.topnav a {
+  float: left;
   display: block;
-  color: #ffffff;
+  color: #fff;
   text-align: center;
   padding: 14px 16px;
   text-decoration: none;
+  font-size: 17px;
+  cursor: pointer;
 }
 
-nav li a:hover {
+.topnav a:hover {
   background-color: #7078BB;
 }
 
-.active {
-  color: #87D870;
+.topnav a.active {
+  color: #4CAF50;
+}
+
+.topnav .icon {
+  display: none;
+}
+
+@media screen and (max-width: 600px) {
+  .topnav a:not(:first-child) {display: none;}
+  .topnav a.icon {
+    float: right;
+    display: block;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .topnav.responsive {position: relative;}
+  .topnav.responsive .icon {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  .topnav.responsive a {
+    float: none;
+    display: block;
+    text-align: left;
+  }
 }
```

Comment:
New line missing 
![image](https://user-images.githubusercontent.com/1935403/98845523-b3340f00-2402-11eb-90a4-59417e01dd25.png)

Reply:
Just curious why we want to surely leave a new line after the main closing in our template files